### ISSUE TITLE
Switch `rem` & `em` `line-height` instances to `px`

### ISSUE THREAD

**Issue by ntwb** [WordPress/twentytwenty#408]
There are 3 instances where `px` are not used for `line-height` in `style.css`, this is a violation of the WordPress CSS Coding Standards

https://developer.wordpress.org/coding-standards/wordpress-coding-standards/css/#values:

> "Line height should also be unit-less, unless necessary to be defined as a specific pixel value. This is more than just a style convention, but is worth mentioning here."
> More information: http://meyerweb.com/eric/thoughts/2006/02/08/unitless-line-heights/

----

• `2264:15  ✖  Unexpected unit "rem" for declaration-property-unit-whitelist property "line-height"`

https://github.com/WordPress/twentytwenty/blob/master/style.css#L2264

```css
.modal-search-form .search-field::-moz-placeholder {
	color: inherit;
	line-height: 8rem;
}
```

• `3008:15  ✖  Unexpected unit "em" for declaration-property-unit-whitelist property "line-height"`

https://github.com/WordPress/twentytwenty/blob/master/style.css#L3008

```css
.wp-block-pullquote::before {
	background: #fff;
	border-radius: 50%;
	color: #cd2653;
	content: "”";
	display: block;
	font-size: 6.2rem;
	font-weight: 500;
	line-height: 1.2em;
	margin: 0 auto 1.5rem auto;
	text-align: center;
	height: 4.4rem;
	width: 4.4rem;
}
```

• `4499:16  ✖  Unexpected unit "rem" for declaration-property-unit-whitelist property "line-height"`

https://github.com/WordPress/twentytwenty/blob/master/style.css#L4499

```css
	.modal-search-form .search-field::-moz-placeholder {
		line-height: 18.3rem;
	}
```

----

The `line-height` should be changed to the correct `px` value for each.

**Comment by ianbelanger79** [comment]
Actually, to meet the latest standards, all line-height values should be unitless moving forward. I converted all of them in core recently.

**Comment by ianbelanger79** [comment]
Fixed in #427
